src/main/java/se/sundsvall/casedata/api/model/validation/enums/MessageType.java

```diff
@@ -0,0 +1,5 @@
+package se.sundsvall.casedata.api.model.validation.enums;
+
+public enum MessageType {
```

Comment:
Is there any drawbacks of reusing the MessageType enum in the "se.sundsvall.casedata.integration.db.model.enums" package?

Reply:
Drawbacks? Maybe not. The intention was to move the enum to the validation package but seems like I just copied it. Will fix